tests/test_TFUtil.py

```diff
@@ -469,6 +469,30 @@ def test_Data_copy_compatible_to_batch_feature_is_dynamic():
   assert_equal(t.get_size_dim_tag(0), dec.get_time_dim_tag())
 
 
+def test_Data_get_common_data_modified_time_dim():
+  d1 = Data(name='t', shape=(None, 32, 128), dtype='float32', auto_create_placeholders=True)
+  d2 = Data(name='r', shape=(None, 32, 128), dtype='float32', auto_create_placeholders=True)
+  # simulating the changes in the time dim
+  d1.size_placeholder[0] -= 1
+  d2.size_placeholder[0] -= 1
```

Comment:
This is not really needed. The time dims of `d1` and `d2` are already different from each other (why do assume that they are the same?).

Reply:
Data constructs tags for the size_placeholders. After the operation (-=1) it's another placeholder. And there's no tag. i.e.
`DimensionTag.get_tag_from_size_tensor(d1.size_placeholder[0])` becomes `None`.

Reply:
I know what it does and what the effect is. I just say, you don't need that. But maybe I misunderstood what situation exactly you want to construct here. But in any case, whatever situation you try to construct (see my other question also), you are doing it too complicated.
Please also add a comment, saying what situation you want to construct (e.g. `# We construct it such that d1 and d2 have different time dim tags.`).

Reply:
I updated the test. Indeed there's no need in modifying them, the error remains.